frontend/src/repository/exams.js

```diff
@@ -19,6 +19,8 @@ export const getExams = async (query = {}, withTS = false) => {
     }
   }
 
+  //const allo = await axios.get('https://teamy.ebm.nymous.io/api/groups');
```

Comment:
allô ?

Reply:
Test for later if we can connect to Teamy api :telephone_receiver:

Reply:
☎️